content/10.zk-stack/20.running/25.gateway-settlement-layer.md

```diff
@@ -11,9 +11,40 @@ enhance **interoperability**, provide **proof aggregation**, and offer **cost ef
 ## Running ZKsync Gateway locally
 
 ::callout{icon="i-heroicons-exclamation-triangle" color="amber"}
-This guide assumes you already have an ecosystem with a ZKsync chain settling on L1 [as detailed in the quickstart](./quickstart).
+This guide assumes you already have `zkstackup` and the required system dependencies installed [as detailed in the quickstart](./quickstart).
 ::
 
+1. Update `foundry-zksync` to use the version from commit `27360d4c8`:
```

Comment:
do we want to have a specific commit in the docs? why is it not merged in the latest release? we're gonna have to update it again once this is merged 🫤

Reply:
What do you mean merged in the latest release? I agree it's not ideal, but for some reason other versions can cause issues, so I'd rather make sure it's working for people. In the zksync-era repo they are using this specific commit in all of their testing: https://github.com/matter-labs/zksync-era/blob/main/docker/zk-environment/Dockerfile#L50